pkg/api/licenses.go

```diff
@@ -45,6 +45,8 @@ import (
 //	@Param			page			query		int						false	"Page number"
 //	@Param			limit			query		int						false	"Limit of responses per page"
 //	@Param			externalRef		query		string					false	"External reference parameters"
+//	@Param			sort_by			query		string					false	"Sort by rf_spdx_id, rf_shortname or rf_fullname"
```

Comment:
Since the possible values for the field is less, recommend putting them as an enum.
```suggestion
//	@Param			sort_by			query		string					false	"Sort by rf_spdx_id, rf_shortname or rf_fullname"	Enums(rf_spdx_id, rf_shortname, rf_fullname)
```